lua/spectre_common.lua

```diff
@@ -319,6 +314,10 @@ local function forward_to_destination(method, request_uri, request_headers)
     }
 end
 
+local function inject_source_header(request)
```

Comment:
Missing docstring. Please mention that we're doing this so that haproxy/envoy knows that this request is from casper and won't re-route it back to it